R/DataSpaceConnection.R

```diff
@@ -596,6 +608,54 @@ DataSpaceConnection <- R6Class(
 
       invisible(NULL)
     },
+
+    .getVirusNameMappingTables = function(){
```

Comment:
Interesting that you are using one method to return a list of three tables instead of having three methods to return three tables each. It's different from how we return data in other methods like `getDataset`, but I suppose this is ok since you written up a clear vignette to use this method. :100: